testreport.txt

```diff
@@ -0,0 +1 @@
izeline\\react-certification-2021\\src\\components\\CardVideo.jsx","statementMap":{"0":{"start":{"line":4,"column":18},"end":{"line":20,"column":1}},"1":{"start":{"line":5,"column":4},"end":{"line":19,"column":5}},"2":{"start":{"line":22,"column":24},"end":{"line":31,"column":1}}},"fnMap":{"0":{"name":"(anonymous_0)","decl":{"start":{"line":4,"column":18},"end":{"line":4,"column":19}},"loc":{"start":{"line":4,"column":64},"end":{"line":20,"column":1}},"line":4}},"branchMap":{},"s":{"0":1,"1":25,"2":1},"f":{"0":25},"b":{},"_coverageSchema":"43e27e138ebf9cfc5966b082cf9a028302ed4184","hash":"0c0f4bc58becfd92898a91ddd548ab8bf06fa874"},"C:\\Users\\Martin Santos Perea\\Documents\\React_Wizeline\\react-certification-2021\\src\\mocks\\youtubevideos.js":{"path":"C:\\Users\\Martin Santos Perea\\Documents\\React_Wizeline\\react-certification-2021\\src\\mocks\\youtubevideos.js","statementMap":{"0":{"start":{"line":1,"column":29},"end":{"line":846,"column":5}}},"fnMap":{},"branchMap":{},"s":{"0":1},"f":{},"b":{},"_coverageSchema":"43e27e138ebf9cfc5966b082cf9a028302ed4184","hash":"76075963907fd09a87513d3c42578dd05d631235"},"C:\\Users\\Martin Santos Perea\\Documents\\React_Wizeline\\react-certification-2021\\src\\index.js":{"path":"C:\\Users\\Martin Santos Perea\\Documents\\React_Wizeline\\react-certification-2021\\src\\index.js","statementMap":{"0":{"start":{"line":7,"column":0},"end":{"line":12,"column":2}}},"fnMap":{},"branchMap":{},"s":{"0":0},"f":{},"b":{},"hash":"90f27260a9da3ed16bceba590ad5e8d1a64d3e57"},"C:\\Users\\Martin Santos Perea\\Documents\\React_Wizeline\\react-certification-2021\\src\\pages\\Home\\Home.page.jsx":{"path":"C:\\Users\\Martin Santos Perea\\Documents\\React_Wizeline\\react-certification-2021\\src\\pages\\Home\\Home.page.jsx","statementMap":{"0":{"start":{"line":7,"column":2},"end":{"line":21,"column":4}},"1":{"start":{"line":11,"column":8},"end":{"line":16,"column":10}}},"fnMap":{"0":{"name":"HomePage","decl":{"start":{"line":5,"column":9},"end":{"line":5,"column":17}},"loc":{"start":{"line":5,"column":20},"end":{"line":22,"column":1}},"line":5},"1":{"name":"(anonymous_1)","decl":{"start":{"line":10,"column":25},"end":{"line":10,"column":26}},"loc":{"start":{"line":11,"column":8},"end":{"line":16,"column":10}},"line":11}},"branchMap":{},"s":{"0":0,"1":0},"f":{"0":0,"1":0},"b":{},"hash":"103acc67cec9dbd86d75722d951b7ca3d5fe1484"},"C:\\Users\\Martin Santos Perea\\Documents\\React_Wizeline\\react-certification-2021\\src\\components\\App\\App.component.jsx":{"path":"C:\\Users\\Martin Santos Perea\\Documents\\React_Wizeline\\react-certification-2021\\src\\components\\App\\App.component.jsx","statementMap":{"0":{"start":{"line":7,"column":2},"end":{"line":18,"column":4}}},"fnMap":{"0":{"name":"App","decl":{"start":{"line":5,"column":9},"end":{"line":5,"column":12}},"loc":{"start":{"line":5,"column":15},"end":{"line":19,"column":1}},"line":5}},"branchMap":{},"s":{"0":0},"f":{"0":0},"b":{},"hash":"1eec0da86d1eb2780b414104b0b122c30c336e9f"}}}
```

Comment:
Normally, the test reports files are ignored from Git, maybe is a good idea to omit this kind of files on the future or if you want, you can check how to remove it from Git, but is ok if is here on this time